### ISSUE TITLE
crash few seconds after loading a big dataset

### ISSUE THREAD

**Issue by Paul-Darius** [yhat/rodeo#555]
Few seconds after I load the datasets given on kaggle for the current "santander product recommendation" competition, rodeo just crashes with no warning.
I load two csv files. One is around 13 million lines long (2.29GB) and an other one is around one million lines long (110.3MB).

**Comment by TakenPilot** [comment]
This is true.  Because Electron's default communication method is IPC, I'm limited by shared memory restrictions.

The fix for this would be changing to WebSockets.
